lib/console_web/controllers/router/device_controller.ex

```diff
@@ -241,6 +242,25 @@ defmodule ConsoleWeb.Router.DeviceController do
               {:ok, %{}}
             end
           end)
+          |> Ecto.Multi.run(:hotspot_stat, fn _repo, %{ event: event, device: device } ->
+            if event.sub_category in ["uplink_confirmed", "uplink_unconfirmed", "join_request"] do
```

Comment:
join_request is not a sub_category, its the category. Join_request's sub_category is actually undefined lol

Reply:
you is correct

Reply:
fixed